Source/WebCore/platform/graphics/ImageBuffer.cpp

```diff
@@ -32,25 +32,64 @@
 #include "GraphicsContext.h"
 #include "HostWindow.h"
 #include "PlatformImageBuffer.h"
+#include <wtf/text/Base64.h>
+
+#if USE(CG)
+#include "ImageBufferUtilitiesCG.h"
+#endif
+#if USE(CAIRO)
+#include "ImageBufferUtilitiesCairo.h"
+#endif
```

Comment:
It is strange that we use platform headers and there is no platform code specific in this file. I think these header files are included because of using platformMIMETypeIsJPEG(). If this is the case I do not think we need to repeat twice in two different header files.

Reply:
These are necessary as data (not named encodeData) is platform specific.

Reply:
This is a good idea. I'll do this in a separate patch.